data/json/vehicles/trains.json

```diff
@@ -249,5 +249,136 @@
+      { "x": -9, "y": -2, "parts": [ "frame", "board" ] },
+      { "x": -9, "y": -1, "parts": [ "frame", "board", "roof" ] },
+      { "x": -9, "y": 0, "parts": [ "frame", "stowboard", "roof" ] },
+      { "x": -9, "y": 1, "parts": [ "frame", "door_sliding", "roof" ] },
+      { "x": -9, "y": 2, "parts": [ "frame", "stowboard", "roof" ] },
+      { "x": -9, "y": 3, "parts": [ "frame", "board", "roof" ] },
+      { "x": -9, "y": 4, "parts": [ "frame", "board" ] },
+      { "x": -10, "y": -2, "parts": [ "frame", "board" ] },
+      { "x": -10, "y": -1, "parts": [ "frame", "aisle", "roof" ] },
+      { "x": -10, "y": 0, "parts": [ "frame", "aisle", "roof" ] },
+      { "x": -10, "y": 1, "parts": [ "frame", "aisle", "roof" ] },
+      { "x": -10, "y": 2, "parts": [ "frame", "aisle", "roof" ] },
+      { "x": -10, "y": 3, "parts": [ "frame", "aisle", "roof" ] },
+      { "x": -10, "y": 4, "parts": [ "frame", "board" ] },
+      { "x": -11, "y": -2, "parts": [ "frame", "windshield", "inboard_mirror" ] },
+      { "x": -11, "y": -1, "parts": [ "frame", "rail_wheel_steerable", "aisle", "roof" ] },
+      { "x": -11, "y": 0, "parts": [ "frame", "aisle", "roof" ] },
+      { "x": -11, "y": 1, "parts": [ "frame", "lit_aisle", "roof" ] },
+      { "x": -11, "y": 2, "parts": [ "frame", "aisle", "roof" ] },
+      { "x": -11, "y": 3, "parts": [ "frame", "rail_wheel_steerable", "aisle", "roof" ] },
+      { "x": -11, "y": 4, "parts": [ "frame", "windshield", "inboard_mirror" ] },
+      { "x": -12, "y": -2, "parts": [ "frame", "board" ] },
+      { "x": -12, "y": -1, "parts": [ "frame", "board", "dashboard", "roof" ] },
+      { "x": -12, "y": 0, "parts": [ "frame", "seat", "dashboard", "roof", "seatbelt" ] },
+      { "x": -12, "y": 1, "parts": [ "frame", "aisle", "cam_control", "dashboard", "roof" ] },
+      { "x": -12, "y": 2, "parts": [ "frame", "seat", "dashboard", "controls", "roof" ] },
+      { "x": -12, "y": 2, "part": "seatbelt" },
+      { "x": -12, "y": 3, "parts": [ "frame", "board", "dashboard", "roof" ] },
+      { "x": -12, "y": 4, "parts": [ "frame", "board" ] },
+      { "x": -13, "y": -2, "parts": [ "frame", "halfboard" ] },
+      { "x": -13, "y": -1, "parts": [ "frame", "board" ] },
+      { "x": -13, "y": 0, "parts": [ "frame", "windshield" ] },
+      { "x": -13, "y": 1, "parts": [ "frame", "windshield" ] },
+      { "x": -13, "y": 2, "parts": [ "frame", "windshield" ] },
+      { "x": -13, "y": 3, "parts": [ "frame", "board" ] },
+      { "x": -13, "y": 4, "parts": [ "frame", "halfboard" ] },
+      { "x": -14, "y": -1, "parts": [ "frame", "halfboard", "rebar_plate" ] },
+      { "x": -14, "y": 0, "parts": [ "frame", "halfboard", "rebar_plate", "headlight" ] },
+      { "x": -14, "y": 1, "parts": [ "frame", "halfboard", "rebar_plate" ] },
+      { "x": -14, "y": 2, "parts": [ "frame", "halfboard", "rebar_plate", "headlight" ] },
+      { "x": -14, "y": 3, "parts": [ "frame", "halfboard", "rebar_plate" ] }
```

Comment:
These rebar plates don't really strike me as something a train would have.

Reply:
and I'd agree with you but I bet you have seen videos of cars standing on rails getting hit by a train - the train never looks even slightly damaged so I felt some armor should be in place and a bunch of steel rods felt like the most fitting after looking at pictures like this: https://observer.com/wp-content/uploads/sites/2/2013/07/acs64.jpg?quality=80&strip

Reply:
I feel that some form of steel plate armor would be part, these rebar plates are basically post-apocalypse armor.

Reply:
And you should probably be using heavy duty frames.

Reply:
oh, I thought steel plate and heavy duty frames were reserved for military use; will do thanks for the feedback

Reply:
another thing to consider: the passenger cart also uses normal steel frames

Reply:
I think it is just as likely that that should be using HD frames.

Reply:
roger that